src/routes/demo.tanstack-query.tsx

```
@@ -1,26 +1,26 @@
-  })
+  });
 
   return (
-    <div className="p-4">
-      <h1 className="text-2xl mb-4">People list</h1>
+    <div className='p-4'>
+      <h1 className='text-2xl mb-4'>People list</h1>
       <ul>
         {data.map((person) => (
-          <li key={person.name}>{person.name}</li>
+          <li name={person.name}>{person.name}</li>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)